pkg/test/local/environment.go

```diff
@@ -0,0 +1,30 @@
+package local
+
+import (
+	"testing"
+	"istio.io/istio/pkg/test"
+)
+
+type LocalEnvironment struct {
```

Comment:
suggestion: rename this to Environment, so that there is no stutter. i.e.
local.localEnvironment v.s. local.Environment.